### PR TITLE
Throttling is sending next within interval

### PR DESCRIPTION
**Pull request by zhigang1992**
Just added tests here to see if I understand `throttling` correctly

### CODE REVIEW

File: ReactiveCocoaTests/Swift/SignalSpec.swift [hunk 1]
```diff
@@ -1058,6 +1058,27 @@ class SignalSpec: QuickSpec {
 				expect(signal).notTo(beNil())
 			}
 
+			it("should not send values when it's less than interval") {
```
Comment (brow) [inline ReactiveCocoaTests/Swift/SignalSpec.swift]: "Should not send values when it's less than interval" seems synonymous with "should send values on the given scheduler at no less than the interval", the name of the next example. It's not clear to me if they are testing distinct properties of `throttle`. Would it make sense to combine the test code into a single example?

**Comment by jspahrsummers** [comment]
It seems like you are expecting `throttle` to delay _all_ values by at least the given interval (like RAC 2.x)—is that correct?

`throttle` no longer behaves that way. Instead, its only role is to ensure that _at least_ the given interval elapses between successive values. If the interval has already elapsed when a value is received, it can be forwarded immediately.

**Comment by zhigang1992** [comment]
@jspahrsummers @brow hm, I see... Guess I should read the doc a bit more careful. :)

**Comment by lingoer** [comment]
:)

**Comment by brow** [comment]
> It seems like you are expecting throttle to delay all values by at least the given interval (like RAC 2.x)—is that correct? throttle no longer behaves that way.

It sounds like it would be appropriate to mention this in the [CHANGELOG](https://github.com/ReactiveCocoa/ReactiveCocoa/blob/master/CHANGELOG.md).

**Comment by brow** [comment]
Also, `enum ThrottleStrategy`, anyone? :trollface:

**Comment by zhigang1992** [comment]
@brow I also found those two
https://github.com/ReactiveCocoa/ReactiveCocoa/issues/1348
https://github.com/ReactiveCocoa/ReactiveCocoa/issues/2075

Maybe debounce would be a good idea

**Pull request closed without merging**